### DISCUSSION TITLE
How to set the history of plane strain deformation?

### DISCUSSION THREAD

**Discussion by siwux** [neperfepx/FEPX#96] [Q&A]
Fepx only allows to set the target load. Can we use it to set the plane strain state and how to do it? 
Thanks a lot in advance.

**Comment by mkasemer** [comment]
@siwux Do you mean for the triaxial case? Yes, we only allow for target loads here, whether enforcing a constant load rate or a constant strain rate. If you are interested in implementation, you could edit both modules to consider strain targets instead.

In uniaxial mode, you can set either target loads or target strains. If you were interested in implementing in triaxial, you could use this mode as a template.
